src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -164,5 +170,34 @@ internal static Settings FromDefaultSources()
 
             return new Settings(configurationSource);
         }
+
+        private static OtlpExportProtocol? GetExporterOtlpProtocol(IConfigurationSource source)
+        {
+            // the defualt in SDK is grpc. http/protobuf should be default for our purposes
+            var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
+
+            if (string.IsNullOrEmpty(exporterOtlpProtocol) || exporterOtlpProtocol == "http/protobuf")
+            {
+                // override settings only for http/protobuf
+                // the second part of the condition is needed to override default endpoint as long as https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868 is not fixed
+                return OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
+            }
+
+            return null;
+        }
+
+        private static Uri GetExporterOtlpEndpoint(IConfigurationSource source, OtlpExportProtocol? otlpExportProtocol)
```

Comment:
consider adding a comment that this would not be needed when https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868 is fixed

Reply:
There is a comment near the `OtlpExportEndpoint` property. I think that it is enough.

Issue created #352